views/check-company.html

```diff
@@ -21,48 +21,57 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             <h1 class="govuk-heading-xl">Check company details</h1>
-            <h2 class="govuk-heading-l">{{scenario.company.name}}</h2>
+            <h2 class="govuk-heading-l">{{company.companyName}}</h2>
 
     {% set inactiveHTML %}
-        {% if scenario.company.status === "Inactive" %}
+        {% if company.companyStatus === "Inactive" %}
             <span class="govuk-!-font-weight-bold">Dissolved</span>
         {% else %}
-            {{scenario.company.status}}
+            {{company.companyStatus}}
         {% endif %}
     {% endset %}
 
     {% set AAoverdueHTML %}
-        {% if scenario.company.accountsStatus === "overdue" %}
-            {{scenario.company.accountsDue}}
+        {% if company.isAccountsOverdue or dueDatePassed %}
+            {{company.accountsDue}}
             <br/>
             <span class="govuk-!-font-weight-bold">Your accounts are overdue</span>
-        {% elseif scenario.company.accountsDue === "inactive" %}
+        {% elseif company.accountsDue === "inactive" %}
             You don’t have to file final accounts with Companies House
         {% else %}
-            {{scenario.company.accountsDue}}
+            {{company.accountsDue}}
         {% endif %}
     {% endset %}
 
     {% set CSoverdueHTML %}
-        {% if scenario.company.csStatus === "overdue" %}
-            {{scenario.company.csDue}}
+        {% if company.isCSOverdue %}
+            {{company.csDue}}
             <br/>
             <span class="govuk-!-font-weight-bold">Your confirmation statement is overdue</span>
-        {% elseif scenario.company.csDue === "inactive" %}
+        {% elseif company.csDue === "inactive" %}
             You don’t have to file final accounts with Companies House
         {% else %}
-            {{scenario.company.csDue}}
+            {{company.csDue}}
         {% endif %}
     {% endset %}
 
-    {% if scenario.company.PTFRequested === "yes" %}
+    {% if company.ptfRequested === "yes" %}
         {% from "warning-text/macro.njk" import govukWarningText %}
         {{ govukWarningText({
         text: "You've already told us you're going to file",
         iconFallbackText: "Warning"
         }) }}
     {% endif %}
 
+
+    {% set address %}
+        {{company.address.line_1}}
+        <br/>
+        {{company.address.line_2}}
+        <br/>
+        {{company.address.postCode}}
+    {% endset %}
+
     {{ govukTable({
```

Comment:
This should be a govukSummaryList element and not a table. While used in the prototype this was deemed not fit for purpose by the DAC report and was subsequently changed to a summary list instead. Have talked to Ollie and he agrees on this. See link below or extensions for the layout.

https://design-system.service.gov.uk/components/summary-list/